pyproject.toml

```diff
@@ -123,3 +123,16 @@ target-version = ['py38']
 [tool.check-wheel-contents]
 # W009: Wheel contains multiple toplevel library entries
 ignore = "W009"
+
+[tool.ruff]
+src = ["pytest"]
+select = [
+    "I",  # isort
+]
+
+[tool.ruff.lint.isort]
```

Comment:
I think [lines-after-imports](https://docs.astral.sh/ruff/settings/#isort-lines-after-imports) = 2 would also be nice (in theory at least...)

Reply:
Let's try it see 16479067c,